### PR TITLE
DOCSP-22433: tutorial change streams

### PR DESCRIPTION
**Pull request by ccho-mongodb**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/10gen/docs-kafka-connector/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-22433>

Staging - https://docs-mongodbcom-staging.corp.mongodb.com/kafka-connector/docsworker-xlarge/DOCSP-22433-tutorial-change-stream/tutorials/explore-change-streams/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [ ] Are all the links working?

### CODE REVIEW

File: source/tutorials/explore-change-streams.txt [hunk 1]
```diff
@@ -0,0 +1,230 @@
+.. _kafka-tutorial-explore-change-streams:
+
```
Comment (biniona-mongodb) [inline source/tutorials/explore-change-streams.txt:2]: Suggestion: Could the right hand on this page TOC be added?

File: source/tutorials/explore-change-streams.txt [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+==============================
+Explore MongoDB Change Streams
+==============================
+
+Follow this tutorial to learn how to create a change stream on a MongoDB
+collection and observe the change events it creates.
+
+If you already understand how to create change streams and the change
+event data they create, proceed to the
```
Comment (biniona-mongodb) [inline source/tutorials/explore-change-streams.txt:11]: Suggestion: Double use of create was a bit hard for me to parse, consider replacing second "create". 

```suggestion
If you already understand how to create change streams and the change
event data they contain, proceed to the
```

File: source/tutorials/explore-change-streams.txt [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+   .. step:: Connect to the Docker Container
+
+      Create two interactive shell sessions on the tutorial Docker
+      Container, each in a separate window.
+
+      .. code-block:: bash
+         :copyable: true
+         :caption: This command starts an interactive shell called shell1
+
+         docker run --rm --name shell1 --network mongodb-kafka-base_localnet -it robwma/mongokafkatutorial:latest bash
```
Comment (biniona-mongodb) [inline source/tutorials/explore-change-streams.txt:29]: Issue I ran into:  Ran into the following error when testing the provided environment. Followed the rest of the tutorial locally and it worked well.

```
docker run --rm --name shell1 --network mongodb-kafka-base_localnet -it robwma/mongokafkatutorial:latest bash
WARNING: The requested image's platform (linux/arm64/v8) does not match the detected host platform (linux/amd64) and no specific platform was requested
docker: Error response from daemon: network mongodb-kafka-base_localnet not found.
```

**Review by biniona-mongodb — APPROVED** [review]
LGTM with comments. Discussed over slack, and in line with discussion I'm omitting feedback around the structure/steps of the tutorial and this tutorials placement in our Docs.

File: source/tutorials/explore-change-streams.txt (hunk 3 above)
Comment (ccho-mongodb) [reply source/tutorials/explore-change-streams.txt:29]: No, the warning is not related to that as it runs ok in emulation. I tested on an intel MBP.

My container is still running from when I tested, but I'll try it again. I'll try to look into the error you're getting.

```
9f8792f7a307   robwma/mongokafkatutorial:latest            "bash"                   3 days ago   Up 3 days                                                                shell1
```

File: source/tutorials/explore-change-streams.txt (hunk 2 above)
Comment (ccho-mongodb) [reply source/tutorials/explore-change-streams.txt:11]: Good catch. I think I will use the verb "produce" as usually you talk about change streams producing or emitting events, not containing them.

**Comment by RWaltersMA** [comment]
In the summary 

"In this tutorial, you created a change stream on MongoDB and observed the output."  

I think we should add a sentence on why they bothered to do this tutorial which had nothing to do with kafka.  Maybe something like

"When you configure the MongoDB Connector for Apache Kafka as a source to a MongoDB cluster, the connection will open a change stream.  In this tutorial, you explored the change stream feature by defining the watch condition and inserting data and observing the change stream event data."

**Comment by ccho-mongodb** [comment]
Merging into https://github.com/mongodb/docs-kafka-connector/pull/2

**Pull request closed without merging**